lib/setup/setupPacketManager.js

```diff
@@ -257,7 +262,7 @@ class PacketManager {
                 return new Promise(resolve =>
                     this._installPackets(packets, () => resolve()));
             })
-            .catch(() => this.logger && this.logger.warn('Some packets could not be installed. Please install them manually'));
+            .catch(() => this.logger && this.logger.warn('Some native dependencies could not be installed. Please install them manually'));
```

Comment:
I would more live to see "packages" ... "native package dependencies"?

Reply:
This was inspired by @AlCalzone
I would also be okay with something like OS dependencies.

Reply:
The question is what it is for the user ... it is a "linux package that he installs via apt" or such ?! Maybe we check in the Telegram channel for naming

Reply:
Currently it can be an apt or a yum package

Reply:
What about "Some <packagemanager-name here> packages could not ..." means we insert the detected package manager here

Reply:
Every single failing package installation will be logged before this message. So the information is already there. It would also be possible to say `Some ${this.mamager} packages could not be installed` Or instead of this.manager printing the operating system.

Reply:
Mehhhhh GitHub does not showed part of my message because I put in „spitze Klammern“. My idea was exactly this.manager

Reply:
Mehhhhh GitHub does not showed part of my message because I put in „spitze Klammern“. My idea was exactly this.manager